apps/ae_socket_connector/lib/socket_connector/application.ex

```diff
@@ -15,6 +15,12 @@ defmodule SocketConnector.Application do
     # See https://hexdocs.pm/elixir/Supervisor.html
     # for other strategies and supported options
     opts = [strategy: :one_for_one, name: SocketConnector.Supervisor]
+
+    Application.get_env(:ae_socket_connector, :node)[:deps_path]
+    |> Path.join("*/ebin")
+    |> Path.wildcard()
+    |> Enum.each(fn path -> Code.append_path(path) end)
```

Comment:
```suggestion
    |> Enum.each(fn path -> true = Code.append_path(path) end)
```
So that we notice if the paths are incorrect